src/auth/auth.service.ts

```diff
@@ -37,23 +42,37 @@ export class AuthService {
   }
 
   // Sign user in by email and password
-  async login(credentials: LoginUserDto): Promise<AccessToken> {
+  async login(response: Response, credentials: LoginUserDto): Promise<AccessToken> {
     const user = await this.validateUser(credentials);
     if(!user) {
       throw new UnauthorizedException();
     }
 
     // Add a username in the JWT token
-    const payload = {
+    const payload: JwtEncodedUserData = {
       sub: user['id'],
       username: user['username'],
       isAdmin: user['isAdmin']
     };
 
-    return {accessToken: await this.jwtService.signAsync(payload)};
+    const jwt = await this.jwtService.signAsync(payload);
+
+    response.cookie(AuthService.ACCESS_COOKIE_NAME, jwt, {
+      httpOnly: true,
+      maxAge: 6 * 60 * 60 * 1000 // 6 hours
```

Comment:
magic number maybe put it in a const file somewhere ?

Reply:
For now, this will stay as is because the env variable is not adaptable to this format